composer.json

```diff
@@ -133,12 +133,12 @@
         "phpstan": "Run PHPStan analysis",
         "phpstan:baseline": "Dump PHPStan baseline file - use only for updating, do not add new errors when possible",
         "post-autoload-dump": "Run additional tasks after installing/updating main dependencies",
-        "qa": "Run QA suite",
+        "qa": "Run QA suite [alias for 'quality-assurance']",
```

Comment:
Why using `[]`? Would `()` be bad for some reason?

Reply:
I'm OK with both and I do not see any objective argument to use one over another - I'm happy with any as long as it's same for all descriptions.

Subjective reason on my side of `[]` over `()` is that I switched to `[]` myself after I broke my hand/wrist, and now using upper side of the keyboard is causing discomfort.

Reply:
The square ones are for default value (is that some convention). I've seen this, that why I've asked:
```console
kub@:~/code/friendsofphp/PHP-CS-Fixer(master)$ php php-cs-fixer --help
Description:
  List commands

Usage:
  list [options] [--] [<namespace>]

Arguments:
  namespace             The namespace name

Options:
      --raw             To output raw command list
      --format=FORMAT   The output format (txt, xml, json, or md) [default: "txt"]
      --short           To skip describing commands' arguments
  -h, --help            Display help for the given command. When no command is given display help for the list command
  -q, --quiet           Do not output any message
  -V, --version         Display this application version
      --ansi|--no-ansi  Force (or disable --no-ansi) ANSI output
  -n, --no-interaction  Do not ask any interactive question
  -v|vv|vvv, --verbose  Increase the verbosity of messages: 1 for normal output, 2 for more verbose output and 3 for debug
```

Reply:
Personally I don't need this kind of information because I believe it's only implementation detail, but if we need to explicitly point it, I would rather go with simpler:

```
"qa": "Alias for 'quality-assurance'"
```

This way full description ("Run QA suite") is defined only in single place, so there's no risk it will get out of sync.